apps/src/calc/blocks.js

```diff
@@ -49,35 +46,11 @@ exports.install = function(blockly, blockInstallOptions) {
 
 };
 
-function initFunctionalBlock(block, title, numArgs) {
-  block.setHSV(184, 1.00, 0.74);
-  block.setFunctional(true, {
-    headerHeight: 30,
-  });
-
-  var options = {
-    fixedSize: { height: 35 },
-    fontSize: 25 // in pixels
```

Comment:
This function actually appears to have been unused originally, note the reference below is to init**Titled**FunctionalBlock